src/elements/common/item/IconCell.js

```diff
@@ -7,6 +7,7 @@ import * as React from 'react';
 import { injectIntl } from 'react-intl';
 import type { IntlShape } from 'react-intl';
 
+import { Archive, FolderArchive } from '@box/blueprint-web-assets/icons/Content';
```

Comment:
can we add a screenshot of these changes and update or add a visual regression test for these ?

Reply:
I've added screenshot to PR description. 

Do you want me to add it to [content explorer README](https://github.com/box/box-ui-elements/blob/master/src/elements/content-explorer/README.md)? Should I replace current screenshot or add new one?

What does BUIE use for visual regression tests? I can't find it